tests/Microsoft.Bot.Builder.Dialogs.Adaptive.Tests/RecognizerSetTests.cs

```diff
@@ -28,5 +75,59 @@ public async Task RecognizerSetTests_None()
         {
             await TestUtils.RunTestScript(_resourceExplorerFixture.ResourceExplorer);
         }
+
+        [Fact]
+        public async Task RecognizerSetTests_Merge_LogsTelemetry_WhenLogPiiTrue()
```

Comment:
Make theory and consolidate, remove `_`

Reply:
THIS IS MY FAVORITE THING. 
Changed `Fact` --> `Theory` and consolidated true/false tests